src/components/index.js

```diff
@@ -10,6 +10,7 @@ export {default as GroupedDropdown} from './inputs/grouped-dropdown'
 export {default as UploadInput} from './inputs/upload-input'
 export {default as UploadInputV2} from './inputs/upload-input-v2'
 export {default as CompanyInput} from './inputs/company-input'
+export {default as CompanyInputV2} from './inputs/company-input-v2.js'
```

Comment:
dont export here
will make the peer dep mandatory check
https://github.com/OpenStackweb/openstack-uicore-foundation/blob/d20ad11c07ff643d7ad0fe6d26e6907aaa3e1133/src/components/index.js#L61